langchain/CHANGELOG.md

```diff
@@ -1 +1,5 @@
 # CHANGELOG - LangChain
+
+***Added***:
```

Comment:
you're missing the date, similar to what we have for airbyte: https://github.com/DataDog/integrations-core/blob/master/airbyte/CHANGELOG.md

Reply:
Which date should the changelog reflect? 

For airbyte, I see the PR to create the integration was merged on 2023-08-01 but the changelog states 2023-07-18

Reply:
I have updated the pr with the merge date of [15229](https://github.com/DataDog/integrations-core/pull/15229)

Reply:
👍